specification/hdinsight/data-plane/Microsoft.HDInsight/preview/2018-11-01-preview/livySpark.json

```diff
@@ -1010,18 +1054,21 @@
       "type": "object",
       "properties": {
         "id": {
+          "description": "Session id.",
```

Comment:
>Session [](start = 26, length = 7)

The "Session" is so common. I think that we had better to add prefix for example Spark Batch, Spark Session, Spark Statement etc. 
Could you please check all the "session" word to make it clear enough?